data/temporal_dataset.py

```diff
@@ -191,6 +215,7 @@ def get_img(
                     mask_square=self.opt.data_online_creation_mask_square_B,
                     crop_dim=self.opt.data_online_creation_crop_size_B,
                     output_dim=self.opt.data_load_size,
+                    context_pixels=self.opt.data_online_context_pixels,
```

Comment:
This change does not seem related to having multiple D ?

Reply:
Actually these errors raised when implementing multiple D because temporal D used to be computed on crop without context even when self.opt.data_online_context_pixels > 0 but it's not directly related to having multiple D.